### ISSUE TITLE
Won't start on wl-roots Wayland (Arch, Hyprland, AMD GPU/CPU)

### ISSUE THREAD

**Issue by Loooser23** [sultim-t/xash-rt#162]
I get the following error when starting the xash3d.exe with Lutris/Bottles/wine: "Host_InitError: Can't initialize any renderer. Check your video drivers!"

I'm running the Hyprland Wayland Compositor with a Ryzen 7 5800X and an RX 6650XT, the driver is the latest amdgpu package in the arch linux kernel.

DXVK and VKD3D are enabled and I have tried Gamescope (often fixes games on Wayland)

I tried wine and ProtonGE.

I'd appreciate help if I missed anything or could try something else. I don't have enough desire to play this to try a different computer or desktop environment, however.

**Comment by A31Nesta** [comment]
I'm having the same issue in X11 and Nvidia (proprietary drivers), also on Arch linux

**Comment by SpidFightFR** [comment]
Same issue for me on VoidLinux with mesa RADV. Drivers are up to date.

**Comment by nonetrix** [comment]
On my AMD GPU on Windows I had to manually update the rtopengl library by replacing the .dll to get the game to work and it worked fine, but tried doing that on Linux and I just get this even when reverting the update to before 25th Anniversary. I am on NixOS with RX 6800 and newer Mesa version that has RT enabled by default  :/
![image](https://github.com/sultim-t/xash-rt/assets/45698918/c4e8256d-e165-45f1-93bd-c940fcc83dbe)
Also here is what happens without updating the .dll just weird artifacts, could be the same white screen bug but just manifests itself differently on Linux
![image](https://github.com/sultim-t/xash-rt/assets/45698918/954df421-6c60-4a09-aef7-d2bcfeaaebc8)

**Comment by SpidFightFR** [comment]
Pretty sure HLRT and DOOM RT are both abandoned projects as of now...

**Comment by nonetrix** [comment]
Yeah but would still like to experiment with them to compare performance, I think the developer ended up getting hired by NVIDIA

**Comment by SpidFightFR** [comment]
> Yeah but would still like to experiment with them to compare performance, I think the developer ended up getting hired by NVIDIA

I'm not really sure NVIDIA works like that.
It's nvidia, not Valve.
